[2024-11-02T14:49:48.774+0000] {processor.py:186} INFO - Started process (PID=1696) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:49:48.776+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:49:48.779+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:49:48.778+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:49:48.823+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:49:49.005+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:49:49.005+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:49:49.045+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:49:49.045+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:49:49.090+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.326 seconds
[2024-11-02T14:50:19.922+0000] {processor.py:186} INFO - Started process (PID=1704) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:19.926+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:50:19.929+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:19.928+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:19.964+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:19.994+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:19.994+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:50:20.043+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:20.042+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:50:20.116+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.208 seconds
[2024-11-02T14:50:50.952+0000] {processor.py:186} INFO - Started process (PID=1713) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:50.955+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:50:50.959+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:50.958+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:51.028+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:51.096+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:51.095+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:50:51.157+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:51.156+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:50:51.199+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.268 seconds
[2024-11-02T14:50:57.453+0000] {processor.py:186} INFO - Started process (PID=1720) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:57.457+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:50:57.463+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:57.462+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:57.595+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:50:57.827+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:57.827+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:50:57.869+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:50:57.869+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:50:57.919+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.497 seconds
[2024-11-02T14:51:01.025+0000] {processor.py:186} INFO - Started process (PID=1721) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:51:01.026+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:51:01.029+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:51:01.028+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:51:01.090+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:51:01.088+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:51:01.097+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:51:01.113+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:51:01.112+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:51:01.150+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:51:01.150+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:51:01.188+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.174 seconds
[2024-11-02T14:51:31.388+0000] {processor.py:186} INFO - Started process (PID=1728) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:51:31.391+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:51:31.396+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:51:31.395+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:51:31.472+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:51:31.470+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:51:31.482+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:51:31.712+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:51:31.712+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:51:31.754+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:51:31.753+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:51:31.797+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.435 seconds
[2024-11-02T14:52:02.722+0000] {processor.py:186} INFO - Started process (PID=1736) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:52:02.725+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:52:02.729+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:52:02.728+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:52:02.800+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:52:02.797+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:52:02.812+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:52:02.870+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:52:02.869+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:52:02.921+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:52:02.921+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:52:02.959+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.261 seconds
[2024-11-02T14:52:33.791+0000] {processor.py:186} INFO - Started process (PID=1744) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:52:33.796+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:52:33.803+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:52:33.802+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:52:33.887+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:52:33.885+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:52:33.892+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:52:33.927+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:52:33.926+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:52:33.971+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:52:33.971+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:52:34.011+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.275 seconds
[2024-11-02T14:53:04.746+0000] {processor.py:186} INFO - Started process (PID=1752) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:53:04.750+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:53:04.753+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:53:04.753+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:53:04.813+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:53:04.811+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:53:04.821+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:53:04.847+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:53:04.847+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:53:04.882+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:53:04.881+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:53:04.921+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.187 seconds
[2024-11-02T14:53:35.409+0000] {processor.py:186} INFO - Started process (PID=1760) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:53:35.413+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:53:35.421+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:53:35.420+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:53:35.509+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:53:35.505+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:53:35.526+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:53:35.642+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:53:35.641+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:53:35.759+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:53:35.758+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:53:35.837+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.455 seconds
[2024-11-02T14:54:06.779+0000] {processor.py:186} INFO - Started process (PID=1768) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:54:06.783+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:54:06.789+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:54:06.788+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:54:06.926+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:54:06.921+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:54:06.952+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:54:07.063+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:54:07.061+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:54:07.131+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:54:07.130+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:54:07.166+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.417 seconds
[2024-11-02T14:54:37.938+0000] {processor.py:186} INFO - Started process (PID=1776) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:54:37.939+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:54:37.941+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:54:37.940+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:54:37.972+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:54:37.971+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:54:37.977+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:54:38.011+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:54:38.010+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:54:38.047+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:54:38.047+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:54:38.083+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.153 seconds
[2024-11-02T14:55:08.349+0000] {processor.py:186} INFO - Started process (PID=1784) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:55:08.353+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:55:08.359+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:55:08.358+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:55:08.424+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:55:08.421+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:55:08.434+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:55:08.489+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:55:08.488+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:55:08.547+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:55:08.546+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:55:08.592+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.271 seconds
[2024-11-02T14:55:39.034+0000] {processor.py:186} INFO - Started process (PID=1792) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:55:39.036+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:55:39.037+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:55:39.037+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:55:39.064+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:55:39.063+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:55:39.070+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:55:39.108+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:55:39.107+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:55:39.146+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:55:39.146+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:55:39.180+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.153 seconds
[2024-11-02T14:56:09.930+0000] {processor.py:186} INFO - Started process (PID=1800) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:56:09.932+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:56:09.935+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:56:09.934+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:56:09.982+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:56:09.981+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:56:09.989+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:56:10.015+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:56:10.015+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:56:10.049+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:56:10.048+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:56:10.089+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.172 seconds
[2024-11-02T14:56:40.924+0000] {processor.py:186} INFO - Started process (PID=1808) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:56:40.928+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:56:40.936+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:56:40.934+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:56:41.043+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:56:41.040+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:56:41.055+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:56:41.093+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:56:41.092+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:56:41.133+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:56:41.133+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:56:41.158+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.261 seconds
[2024-11-02T14:57:11.909+0000] {processor.py:186} INFO - Started process (PID=1816) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:57:11.911+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:57:11.915+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:57:11.914+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:57:11.943+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:57:11.942+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:57:11.948+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:57:11.977+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:57:11.976+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:57:12.019+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:57:12.018+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:57:12.055+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.159 seconds
[2024-11-02T14:57:42.646+0000] {processor.py:186} INFO - Started process (PID=1824) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:57:42.650+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:57:42.653+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:57:42.652+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:57:42.713+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:57:42.711+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:57:42.723+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:57:42.769+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:57:42.769+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:57:42.800+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:57:42.800+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:57:42.828+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.210 seconds
[2024-11-02T14:58:13.642+0000] {processor.py:186} INFO - Started process (PID=1832) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:58:13.646+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:58:13.651+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:58:13.650+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:58:13.711+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:58:13.708+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:58:13.721+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:58:13.749+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:58:13.749+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:58:13.777+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:58:13.777+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:58:13.812+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.197 seconds
[2024-11-02T14:58:44.518+0000] {processor.py:186} INFO - Started process (PID=1840) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:58:44.520+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:58:44.524+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:58:44.523+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:58:44.578+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:58:44.576+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:58:44.588+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:58:44.617+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:58:44.617+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:58:44.653+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:58:44.653+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:58:44.685+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.184 seconds
[2024-11-02T14:59:15.264+0000] {processor.py:186} INFO - Started process (PID=1848) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:59:15.268+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:59:15.272+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:59:15.271+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:59:15.327+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:59:15.325+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:59:15.335+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:59:15.373+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:59:15.372+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:59:15.408+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:59:15.407+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:59:15.436+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.201 seconds
[2024-11-02T14:59:46.229+0000] {processor.py:186} INFO - Started process (PID=1856) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:59:46.233+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T14:59:46.237+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:59:46.236+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:59:46.294+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:59:46.292+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T14:59:46.305+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T14:59:46.340+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:59:46.340+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T14:59:46.370+0000] {logging_mixin.py:190} INFO - [2024-11-02T14:59:46.370+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T14:59:46.415+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.211 seconds
[2024-11-02T15:00:16.625+0000] {processor.py:186} INFO - Started process (PID=1864) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:00:16.627+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:00:16.630+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:00:16.629+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:00:16.664+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:00:16.661+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:00:16.672+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:00:16.698+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:00:16.698+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:00:16.727+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:00:16.727+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:00:16.754+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.139 seconds
[2024-11-02T15:00:47.565+0000] {processor.py:186} INFO - Started process (PID=1872) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:00:47.568+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:00:47.576+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:00:47.575+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:00:47.635+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:00:47.631+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:00:47.644+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:00:47.683+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:00:47.682+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:00:47.718+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:00:47.717+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:00:47.743+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.205 seconds
[2024-11-02T15:01:18.415+0000] {processor.py:186} INFO - Started process (PID=1880) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:01:18.416+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:01:18.418+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:01:18.417+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:01:18.448+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:01:18.447+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:01:18.454+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:01:18.482+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:01:18.482+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:01:18.514+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:01:18.514+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:01:18.547+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.140 seconds
[2024-11-02T15:01:49.353+0000] {processor.py:186} INFO - Started process (PID=1887) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:01:49.354+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:01:49.356+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:01:49.355+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:01:49.384+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:01:49.383+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:01:49.392+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:01:49.427+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:01:49.427+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:01:49.470+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:01:49.469+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:01:49.509+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.164 seconds
[2024-11-02T15:02:20.280+0000] {processor.py:186} INFO - Started process (PID=1895) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:02:20.282+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:02:20.288+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:02:20.286+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:02:20.354+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:02:20.352+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:02:20.360+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:02:20.389+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:02:20.389+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:02:20.418+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:02:20.418+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:02:20.447+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.184 seconds
[2024-11-02T15:02:51.207+0000] {processor.py:186} INFO - Started process (PID=1903) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:02:51.209+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:02:51.212+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:02:51.212+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:02:51.242+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:02:51.241+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:02:51.248+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:02:51.274+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:02:51.274+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:02:51.320+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:02:51.319+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:02:51.355+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.161 seconds
[2024-11-02T15:03:22.144+0000] {processor.py:186} INFO - Started process (PID=1911) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:03:22.149+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:03:22.155+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:03:22.154+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:03:22.301+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:03:22.297+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:03:22.317+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:03:22.353+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:03:22.353+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:03:22.385+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:03:22.384+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:03:22.420+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.305 seconds
[2024-11-02T15:03:52.889+0000] {processor.py:186} INFO - Started process (PID=1919) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:03:52.890+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:03:52.893+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:03:52.892+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:03:52.930+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:03:52.929+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:03:52.938+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:03:52.969+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:03:52.969+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:03:53.003+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:03:53.002+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:03:53.038+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.163 seconds
[2024-11-02T15:04:23.838+0000] {processor.py:186} INFO - Started process (PID=1928) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:04:23.840+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:04:23.845+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:04:23.844+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:04:23.901+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:04:23.899+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:04:23.908+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:04:23.939+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:04:23.939+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:04:23.974+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:04:23.973+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:04:24.001+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.197 seconds
[2024-11-02T15:04:54.203+0000] {processor.py:186} INFO - Started process (PID=1936) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:04:54.207+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:04:54.212+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:04:54.211+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:04:54.270+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:04:54.267+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:04:54.278+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:04:54.304+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:04:54.303+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:04:54.334+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:04:54.333+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:04:54.370+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.192 seconds
[2024-11-02T15:05:25.249+0000] {processor.py:186} INFO - Started process (PID=1944) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:05:25.254+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:05:25.260+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:05:25.259+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:05:25.376+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:05:25.371+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:05:25.397+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:05:25.512+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:05:25.511+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:05:25.559+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:05:25.559+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:05:25.600+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.394 seconds
[2024-11-02T15:05:56.256+0000] {processor.py:186} INFO - Started process (PID=1952) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:05:56.258+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:05:56.259+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:05:56.259+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:05:56.291+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:05:56.289+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:05:56.297+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:05:56.327+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:05:56.326+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:05:56.361+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:05:56.360+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:05:56.400+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.154 seconds
[2024-11-02T15:06:27.165+0000] {processor.py:186} INFO - Started process (PID=1966) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:06:27.167+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:06:27.168+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:06:27.168+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:06:27.204+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:06:27.203+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:06:27.210+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:06:27.236+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:06:27.236+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:06:27.268+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:06:27.267+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:06:27.300+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.143 seconds
[2024-11-02T15:06:58.035+0000] {processor.py:186} INFO - Started process (PID=1974) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:06:58.036+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:06:58.038+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:06:58.037+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:06:58.072+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:06:58.070+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:06:58.078+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:06:58.105+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:06:58.105+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:06:58.134+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:06:58.134+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:06:58.170+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.142 seconds
[2024-11-02T15:07:29.002+0000] {processor.py:186} INFO - Started process (PID=1982) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:07:29.004+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:07:29.010+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:07:29.009+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:07:29.080+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:07:29.077+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:07:29.092+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:07:29.127+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:07:29.127+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:07:29.170+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:07:29.169+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:07:29.198+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.215 seconds
[2024-11-02T15:08:00.008+0000] {processor.py:186} INFO - Started process (PID=1990) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:08:00.012+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:08:00.022+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:08:00.021+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:08:00.066+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:08:00.065+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:08:00.073+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:08:00.105+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:08:00.104+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:08:00.136+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:08:00.135+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:08:00.164+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.183 seconds
[2024-11-02T15:08:30.919+0000] {processor.py:186} INFO - Started process (PID=1998) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:08:30.921+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:08:30.925+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:08:30.924+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:08:30.960+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:08:30.959+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:08:30.966+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:08:30.993+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:08:30.993+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:08:31.024+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:08:31.024+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:08:31.066+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.162 seconds
[2024-11-02T15:09:01.817+0000] {processor.py:186} INFO - Started process (PID=2006) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:09:01.819+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:09:01.821+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:09:01.821+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:09:01.857+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:09:01.855+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:09:01.869+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:09:01.921+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:09:01.920+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:09:01.955+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:09:01.955+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:09:01.993+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.186 seconds
[2024-11-02T15:09:32.814+0000] {processor.py:186} INFO - Started process (PID=2014) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:09:32.818+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:09:32.824+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:09:32.823+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:09:32.934+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:09:32.931+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:09:32.946+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:09:32.996+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:09:32.996+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:09:33.031+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:09:33.031+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:09:33.060+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.271 seconds
[2024-11-02T15:10:03.257+0000] {processor.py:186} INFO - Started process (PID=2023) to work on /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:10:03.259+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/templatin/tem.py for tasks to queue
[2024-11-02T15:10:03.263+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:10:03.262+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:10:03.341+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:10:03.338+0000] {templater.py:94} ERROR - Failed to resolve template field 'bash_command'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/template/templater.py", line 92, in resolve_template_files
    setattr(self, field, env.loader.get_source(env, content)[0])  # type: ignore
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/jinja2/loaders.py", line 207, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /opt/airflow/dags/templatin/wiki.sh
[2024-11-02T15:10:03.353+0000] {processor.py:925} INFO - DAG(s) 'bas' retrieved from /opt/airflow/dags/templatin/tem.py
[2024-11-02T15:10:03.414+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:10:03.413+0000] {dag.py:3229} INFO - Sync 1 DAGs
[2024-11-02T15:10:03.486+0000] {logging_mixin.py:190} INFO - [2024-11-02T15:10:03.485+0000] {dag.py:4156} INFO - Setting next_dagrun for bas to 2024-11-02 00:00:00+00:00, run_after=2024-11-03 00:00:00+00:00
[2024-11-02T15:10:03.546+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/templatin/tem.py took 0.304 seconds
